app/src/main/res/layout/activity_study_burst.xml

```diff
@@ -0,0 +1,226 @@
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+            <Button
+                android:id="@+id/studyBurstBack"
+                android:layout_width="30dp"
+                android:layout_height="30dp"
+                android:layout_margin="8dp"
+                android:background="@drawable/rs2_cancel_icon"/>
+            <ProgressBar
+                android:id="@+id/studyBurstTopProgressBar"
+                style="?android:attr/progressBarStyleHorizontal"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="0dp"
+                android:layout_marginRight="20dp"
+                android:layout_gravity="center_vertical"
+                android:indeterminate="false"
+                android:max="4"
+                android:progress="2"
+                android:scaleY="0.65"
+                android:progressDrawable="@drawable/progress_drawable"/>
+        </LinearLayout>
+        <TextView
+            android:id="@+id/studyBurstTitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textColor="@color/white"
+            android:textSize="24sp"
+            android:fontFamily="@font/lato_bold"
+            android:layout_margin="10dp"
+            android:gravity="center"/>
+        <TextView
+            android:id="@+id/studyBurstMessage"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textColor="@color/white"
+            android:textSize="18sp"
+            android:fontFamily="@font/lato"
+            android:layout_margin="20dp"
+            android:gravity="center"/>
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/middle_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_margin="10dp"
+        app:layout_constraintTop_toBottomOf="@id/top_container"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent">
+        <View
+            android:layout_width="0dp"
+            android:layout_height="2dp"
+            android:layout_gravity="center_vertical"
+            android:layout_weight="2"
+            android:background="@color/appLightGray"/>
+        <org.sagebionetworks.research.mpower.studyburst.StudyBurstStatusWheel
+            android:id="@+id/studyBurstStatusWheel"
+            android:layout_width="70dp"
+            android:layout_height="70dp"
+            android:layout_marginLeft="20dp"
+            android:layout_marginRight="20dp">
+
+        </org.sagebionetworks.research.mpower.studyburst.StudyBurstStatusWheel>
+        <!--android.support.constraint.ConstraintLayout

```

Comment:
Smart idea encapsulating this view.  Do you still want this commented code?

Reply:
Nope, it will be gone in the next commit.